target/linux/kirkwood/config-5.4

```diff
@@ -1,22 +1,6 @@
 CONFIG_ALIGNMENT_TRAP=y
 CONFIG_ARCH_32BIT_OFF_T=y
 CONFIG_ARCH_CLOCKSOURCE_DATA=y
-CONFIG_ARCH_HAS_BINFMT_FLAT=y
```

Comment:
Should be kernel refresh different patch?

Reply:
This seams to be the result of a8fb12a7d62c and I feel it deserves a separate patch.
Or do you mean consolidating two refresh commits?

Reply:
So, it's treewide. Maybe it should be done for all targets in one commit...

@adschm 
What is Your opinion about it?

Reply:
Hmm... It seems I need to build a whole toolchain to run `make kernel_oldconfig` and it's out of my scope to do that for every target...

Reply:
I'm not sure what's exactly the question, as the general refresh removing the ARCH symbols is already in separate commits.

But I think it's good to have them separate indeed.